forch/forchestrator.py

```diff
@@ -345,8 +361,10 @@ def _process_device_behavior(self, mac, device_behavior, static=False):
     def handle_auth_result(self, mac, access, segment, role):
         """Method passed as callback to authenticator to forward auth results"""
         self._faucet_collector.update_radius_result(mac, access, segment, role)
-        device_behavior = DeviceBehavior(segment=segment, role=role)
-        self._port_state_manager.handle_device_behavior(mac, device_behavior)
+        with self._lock:
+            if not self._should_ignore_auth_result:
```

Comment:
add warning log here that authentication ignored for this particular device

Reply:
Done